tests/django/test_django_request_no_middleware.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from fastapi import FastAPI
+from fastapi.django import DjangoRequestDep
+from fastapi.testclient import TestClient
+
+app = FastAPI()
+
+
+@app.get("/")
+async def django_user(django_request: DjangoRequestDep):
+    user = django_request.user
+
+    if not user.is_authenticated:
+        return {"error": "User not authenticated"}
+
+    return {"username": user.username}
+
+
+client = TestClient(app)
+
+
+def test_returns_an_error():
+    with pytest.raises(ValueError, match="Django Request not found"):
+        client.get("/")
```

Comment:
I wanted to give a hint if the user forgot to add the middleware, not sure if this is the best way though 😊